README.md

```diff
@@ -34,29 +33,18 @@ After working on your contribution, check for [accessibility violations](#testin
 If Gatsby needs to run on local IP for testing on other machines or devices use `npm run dev:expose` which sets host to `0.0.0.0`.
 If you want to set host to a specific IP address for example `172.17.12.1` run `npm run dev -H 172.17.12.1`.
 
-### Create a new component
+### Create components, layouts, utilities, or demos
```

Comment:
This looks good. I think we could probably make it a little more reader friendly. Maybe something like...

```
### Create components, layouts...

To create source file scaffolding for a new component, layout, utility, or demo, run the NPM script:

`npm run generate <CamelName>`

Below are the full options for this script:

```sh
Options:
  -f, --folder <folder>  Source folder (components, demos, layouts, or utilities) (default: "components")
```

#### Examples

To create a "Test component" component (`.pf-c-test-component`), run:

`npm run generate <TestComponent>`

To create a "Test layout" layout (`.pf-l-test-layout`), run:

`npm run generate <TestLayout> -f layouts`

To create 3 new demos named "Test demo", "Test demo 2", and "Test demo 3", run:

`npm run generate <TestDemo> <TestDemo2> <TestDemo3> -f demos`

Reply:
if I run `npm run generate -f layouts FooBarBar` it creates `src/components/layouts` and `src/components/FooBarBar`

Reply:
Ugh, this is due to the way `npm run` captures the `-f layouts`. This is what it's running under the hood `node scripts/generate "layouts" "FooBarBar"`.

I'll make the script executable and change the docs to call the script directly like `./generate -f layouts FooBarBar` to avoid `npm run`.